### PR TITLE
Amino counts

### PR DESCRIPTION
**Pull request by mehradans92**
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/User/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/User/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

dataset:   Hemo |   |   |   |   |   | dataset: Sol |   |  
-- | -- | -- | -- | -- | -- | -- | -- | --
Model descriptions | AUC | ACC |   |   |   | Model descriptions | AUC | ACC
Baseline counts fr | 0.797 ± 0.049 | 0.819 ± 0.03 |   |   |   | Baseline counts fr | 0.733 ± 0.021 | 0.666 ± 0.019
Baseline counts | 0.8 ± 0.04 | 0.81 ± 0.047 |   |   |   | baseline_counts_fr_single_lstm | 0.742 ± 0.019 | 0.689 ± 0.024
Baseline 3 counts | 0.797 ± 0.039 | 0.809 ± 0.039 |   |   |   | baseline_counts_fr_no_LayerNormalization | 0.709 ± 0.023 | 0.666 ± 0.018
Baseline 3 counts fr | 0.802 ± 0.019 | 0.816 ± 0.027 |   |   |   | baseline_counts_fr_no_LayerNormalization_dropout | 0.727 ± 0.013 | 0.663 ± 0.012
Baseline no counts | 0.781 ± 0.030 | 0.811 ± 0.025 |   |   |   | Baseline no counts | 0.715 ± 0.020 | 0.652 ± 0.025
Baseline counts no-sigmoid | 0.786 ± 0.025 | 0.84 ± 0.027 |   |   |   | baseline_counts_fr_to final_layer_only | 0.729 ± 0.023 | 0.670 ± 0.018
Baseline counts fr no-sigmoid | 0.795 ± 0.018 | 0.819 ± 0.029 |   |   |   | baseline_counts_fr_removed_second_dens_group | 0.738 ± 0.026 | 0.688 ± 0.017
baseline_counts_fr_single_lstm | 0.807 ± 0.035 | 0.818 ± 0.038 |   |   |   | baseline_counts_fr_removed_first_and_second_dens_group | 0.723 ± 0.019 | 0.661 ± 0.022
baseline_counts_fr_to   final_layer_only | 0.779 ± 0.035 | 0.814 ± 0.044 |   |   |   |   |   |  
baseline_counts_fr_no_LayerNormalization | 0.804 ± 0.033 | 0.819 ± 0.036 |   |   |   |   |   |  
baseline_counts_fr_no_LayerNormalization_dropout | 0.791 ± 0.031 | 0.813 ± 0.024 |   |   |   |   |   |  
baseline_counts_fr_removed_second_dens_group | 0.783 ± 0.030 | 0.809 ± 0.04 |   |   |   |   |   |  
baseline_counts_fr_removed_first_and_second_dens_group | 0.805 ± 0.046 | 0.818 ± 0.031 |   |   |   |   |   |  

</body>

</html>

### CODE REVIEW

File: ml/CV/Hemo_RNN_CV.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+shuffled_labels = labels[i]
+shuffled_features = features[i]
+ 
+L = None#features[0].shape[-1]
+N = len(shuffled_labels)
+
+from sklearn.model_selection import ShuffleSplit
+kf = KFold(n_splits=5, shuffle=True)
+folds_results = []
+for i, (train_index, test_index) in enumerate(kf.split(shuffled_features, shuffled_labels)):
```
Comment (mehradans92) [inline ml/CV/Hemo_RNN_CV.py:138]: @whitead This is the CV part. The rest of the script is quite similar to what is already in the notebooks.

**Comment by whitead** [comment]
@mehradans92 are you going to edit js here or another PR?

**Comment by mehradans92** [comment]
Yes. Not done yet

**Comment by mehradans92** [comment]
@whitead Models seems to be loaded from master in `tf-models.js`. I am gonna merge this to master and update js code in another pr.
